addons/point_of_sale/static/src/app/generic_components/numpad/numpad.js

```diff
@@ -19,7 +20,17 @@ export const buttonsType = {
         String,
     ],
 };
-export function getButtons(env, rightColumn) {
+
+export const DECIMAL = {
+    get value() {
+        return localization.decimalPoint;
+    },
+};
```

Comment:
```
export const DECIMAL = { value: localization.decimalPoint };
```
this doesn't work.

Reply:
but my point was that you don't have to do any of this. You can just call `localization.decimalPoint` where you need it

Reply:
oh, I see.

Reply:
Check the last commit. I doesn't work.

Reply:
ok :+1: